src/PowerShell/Microsoft.WinGet.Client/Microsoft.WinGet.Client.psd1

```diff
@@ -66,11 +66,14 @@ PowerShellVersion = '5.1.0'
 FormatsToProcess = 'Format.ps1xml'
 
 # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-NestedModules = if ($env:PROCESSOR_ARCHITECTURE -like 'amd64') {
-    "x64\$PSEdition\Microsoft.WinGet.Client.dll"
+NestedModules = if ($env:PROCESSOR_ARCHITECTURE -like 'X86') {
+    "x86\$PSEdition\Microsoft.WinGet.Client.dll"
+}
+elseif ($env:PROCESSOR_ARCHITECTURE -like 'ARM64') {
+    "ARM64\$PSEdition\Microsoft.WinGet.Client.dll"
 }
 else {
-    "x86\$PSEdition\Microsoft.WinGet.Client.dll"
+    "x64\$PSEdition\Microsoft.WinGet.Client.dll"
```

Comment:
>x64 [](http://example.com/codeflow?start=5&length=3)

x86 has most compatibility, so I think previous logic of making x86 as default makes sense